modules/core/04-channel/keeper/timeout_test.go

```diff
@@ -247,46 +248,120 @@ func (suite *KeeperTestSuite) TestTimeoutPacket() {
 }
 
 // TestTimeoutExectued verifies that packet commitments are deleted on chainA after the
-// channel capabilities are verified.
+// channel capabilities are verified. In addition, the test verified that the channel state
+// after a timeout is updated accordingly.
 func (suite *KeeperTestSuite) TestTimeoutExecuted() {
 	var (
 		path    *ibctesting.Path
 		packet  types.Packet
 		chanCap *capabilitytypes.Capability
 	)
 
-	testCases := []testCase{
-		{"success ORDERED", func() {
-			path.SetChannelOrdered()
-			suite.coordinator.Setup(path)
+	testCases := []struct {
+		msg       string
+		malleate  func()
+		expResult func(packetCommitment []byte, err error)
+	}{
+		{
+			"success ORDERED",
+			func() {
+				path.SetChannelOrdered()
+				suite.coordinator.Setup(path)
+
+				timeoutHeight := clienttypes.GetSelfHeight(suite.chainB.GetContext())
```

Comment:
i think there's also a default timeout height we could use `suite.chainB.GetTimeoutHeight()`

Reply:
ah yeah, works out with the suggestion. I'm seeing the rest of the tests here use the old version so might be better to open an issue and change this altogether in a later point? wdyt?

Reply:
Definitely works for me!